src/components/molecules/LocationMap.tsx

```diff
@@ -0,0 +1,24 @@
+import 'leaflet-css';
+import React, { FunctionComponent } from 'react';
+import { Map, Marker, TileLayer } from 'react-leaflet';
+
+interface IProps {
+    marker: { lat: number, lng: number }
+};
+
+const INITIAL_ZOOM = 13;
+const WRAPPER_STYLES = { height: '100%', width: '100%' };
+
+const LocationMap: FunctionComponent<IProps> = ({ marker }) => {
+    return (
+        <Map center={marker} zoom={INITIAL_ZOOM} style={WRAPPER_STYLES}>
```

Comment:
I've tried it locally and it didn't work.
`position` for `   <Marker position={position}>` should be of type `LatLngTuple`, like: 
```
  const position =[marker.lat, marker.lng] as LatLngTuple;
```
See [react-leaflet simple example](https://github.com/PaulLeCam/react-leaflet/blob/master/example/components/simple.js)
Also, I coudn't see any marker even after this change. I believe it's have to do with [this issue](https://github.com/PaulLeCam/react-leaflet/issues/453]

Reply:
Thanks for the marker bug research!

As for the type, Leaflet supports a few coordinates structures, including this one.